### ISSUE TITLE
No Response

### ISSUE THREAD

**Issue by SparkyRedcar** [louy/homebridge-salus-it500#2]
When using in the HomeKit app I seem to get poor connection with Salus, often shows “No Response”

It sort of works but is very unstable.

**Comment by louy** [comment]
Yeah I’ve had the same. I think this happens when the session expires and the plug-in is trying to reconnect. I might need to try to keep the session open indefinitely to fix it but I need to test that first

**Comment by SparkyRedcar** [comment]
Ahhh no worries. Thanks for your work, let me know if you have any success.
